packages/Jinja2/meta.yaml

```diff
@@ -10,3 +12,8 @@ source:
 test:
   imports:
     - jinja2
```

Comment:
I think when tested imports matches `top-level` (probably most cases), we should remove `test.imports` and fallback to `package.top-level` to avoid redundant information in the `meta.yaml`

Reply:
Sounds good to me